instance/migrations/0025_auto_20150920_0907.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import instance.models.utils
+import django.db.models.fields
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instance', '0024_auto_20150911_2304'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='GeneralLogEntry',
+            fields=[
+                ('id', models.AutoField(serialize=False, auto_created=True, verbose_name='ID', primary_key=True)),
+                ('created', django_extensions.db.fields.CreationDateTimeField(default=django.db.models.fields.NOT_PROVIDED, verbose_name='created', auto_now_add=True)),
+                ('modified', django_extensions.db.fields.ModificationDateTimeField(auto_now=True, default=django.db.models.fields.NOT_PROVIDED, verbose_name='modified')),
```

Comment:
@antoviaque All LogEntry models have a `modified` field. Do we ever expect log entries to be modified after they are created? I would expect log entries to be immutable.